<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbl_titulo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Escolha a opção com o qual deseja prosseguir:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_barra_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-60</y>
     <width>20</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../imagens/faixa.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_barra">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>-60</y>
     <width>20</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../imagens/faixa.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_logo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>500</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../imagens/logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_entrar_copy_arquivos">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
border-radius: 10px;
background-color: rgb(118, 118, 118);</string>
   </property>
   <property name="text">
    <string>Copiar somente arquivos</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_entrar_copy_pastas">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(118, 118, 118);
color: white;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Copiar arquivos e pastas</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sair">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>490</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;
color: white;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_entrar_del_pastas">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(118, 118, 118);
color: white;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Deletar somente pastas</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_entrar_del_arquivos">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
border-radius: 10px;
background-color: rgb(118, 118, 118);</string>
   </property>
   <property name="text">
    <string>Deletar somente arquivos</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_entrar_ren_pastas">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(118, 118, 118);
color: white;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Renomear uma pasta</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_entrar_ren_arquivos">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
border-radius: 10px;
background-color: rgb(118, 118, 118);</string>
   </property>
   <property name="text">
    <string>Renomear um arquivo</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btn_entrar_copy_arquivos</tabstop>
  <tabstop>btn_entrar_copy_pastas</tabstop>
  <tabstop>btn_entrar_del_arquivos</tabstop>
  <tabstop>btn_entrar_del_pastas</tabstop>
  <tabstop>btn_entrar_ren_arquivos</tabstop>
  <tabstop>btn_entrar_ren_pastas</tabstop>
  <tabstop>btn_sair</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
